std/stdio.d

```diff
@@ -2226,6 +2298,42 @@ $(XREF file,readText)
         assert(blc.front is blc.front);
     }
 
+    /**
+    Parse from the file handle one line at a time into a record tuple.
```

Comment:
I think this should be: "Creates an input range set up to parse lines from the file into a tuple." (Note creates, not create). Using _file handle_ could be confused with `FILE`.